docker/JBoss/pom.xml

```diff
@@ -14,8 +14,8 @@
 	</parent>
 
 	<properties>
-		<config.image.repository>nexus.frankframework.org/private</config.image.repository>
-		<config.image.name>iaf-test</config.image.name>
+		<config.image.repository>private.docker.nexus.frankframework.org</config.image.repository>
```

Comment:
Kan je de repo naam niet beter in de docker (hoofd) pom zetten?

Reply:
Ja, alles wat overeenkomt, zal ik in de pom van de docker module zetten, voor frank2example en frank-framework zal ik deze overschrijven.